vscode-trace-extension/src/trace-explorer/available-views/trace-explorer-available-views-webview-provider.ts

```diff
@@ -36,6 +36,10 @@ export class TraceExplorerAvailableViewsProvider implements vscode.WebviewViewPr
         }
     }
 
+    public rerender() {
```

Comment:
I think you can call the lines below in method `updateTraceServerUrl()`. With this you can avoid adding this public API.